### PR TITLE
feat(babel-preset-sui): add preact and flow support

### PR DESCRIPTION
**Pull request by miduga**
Add Preact and Flow support to babel preset.

## Description
Make Preact compatible by checking if Preact is installed instead of React and load the needed stuff for make it work. Add Flow support as well, checking if flow-bin is installed in order to add the preset to strip the flow stuff when compiling.

## Related Issue
https://github.com/SUI-Components/sui/issues/104

### CODE REVIEW

File: packages/babel-preset-sui/lib/is-installed.js [hunk 1]
```diff
@@ -1,9 +1,15 @@
-module.exports = function (base, module) {
-  var isInstall = false
-  try {
-    require(base)
-    isInstall = true
-  } catch (e) {}
+module.exports = function (packagesToTest, moduleToRequire) {
+  // check if the param is an array, if not, convert to it
+  packagesToTest = Array.isArray(packagesToTest) ? packagesToTest : [packagesToTest]
```
Comment (carlosvillu) [inline packages/babel-preset-sui/lib/is-installed.js]: friki comment:

packagesToTest = [...packagesToTest] :P

File: packages/babel-preset-sui/package.json [hunk 2]
```diff
@@ -9,16 +9,18 @@
...
-    "babel-plugin-transform-react-remove-prop-types": "0.4.6",
+    "babel-plugin-transform-object-rest-spread": "6.26.0",
+    "babel-plugin-transform-react-remove-prop-types": "0.4.8",
     "babel-plugin-transform-runtime": "6.23.0",
-    "babel-preset-es2015": "6.22.0",
-    "babel-preset-react": "6.23.0",
+    "babel-preset-es2015": "6.24.1",
+    "babel-preset-flow": "6.23.0",
+    "babel-preset-react": "6.24.1",
+    "flow-bin": "^0.54.0",
```
Comment (carlosvillu) [inline packages/babel-preset-sui/package.json]: ^?!

Comment (miduga) [reply packages/babel-preset-sui/package.json]: Hostia, node 8 te mete el save a piñon en el package.json si haces un `npm install dependency`

File: packages/babel-preset-sui/lib/is-installed.js (hunk 1 above)
Comment (miduga) [reply packages/babel-preset-sui/lib/is-installed.js]:
```
let packagesToTest = 'react'
packagesToTest = [...packagesToTest] 
console.log(packagesToTest) // ['r', 'e', 'a', 'c', 't']
```

Comment (carlosvillu) [reply packages/babel-preset-sui/lib/is-installed.js]: Ohhh sure, an string is an iterable :(

Comment (miduga) [reply packages/babel-preset-sui/lib/is-installed.js]: hahaha :+1: Nice anyway!

Comment (miduga) [reply packages/babel-preset-sui/lib/is-installed.js]: Fixed! :)

**Pull request merged**